app/scripts/directives/overview/serviceInstanceRow.js

```diff
@@ -25,7 +28,9 @@
                               AuthorizationService,
                               BindingService,
                               ListRowUtils,
-                              ServiceInstancesService) {
+                              ServiceInstancesService,
```

Comment:
They have a standard of keeping these in alphabetical order same with the above list